### PR TITLE
O3-1937: Write E2E test for Patient List

### PR DESCRIPTION
**Pull request by Piumal1999**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

Created the following E2E Tests for esm-patient-list app. 
- Creating and editing a patient list
- Deleting a patient list
- Managing patients in a patient list

Related Doc: https://docs.google.com/document/d/1qD_zpYUG8C28zoAhrl7slI55lInyO7a-QR3tiwhrjwk/edit#heading=h.swr4izgpg4my 

## Screenshots
image

## Related Issue

https://issues.openmrs.org/projects/O3/issues/O3-1937

## Other

⚠️ Do not merge this PR or run the tests until [COH-51 ](https://issues.openmrs.org/browse/COH-51) is fixed. Otherwise, it will fail to delete the created patient lists and cohort types.

Update: This bug is fixed and the PR is ready for review

### CODE REVIEW

**Comment by Piumal1999** [comment]
Finally, this is ready for merging. @jayasanka-sack @anjula-sack Could you please review and merge this PR?

File: packages/esm-patient-list-app/src/create-edit-patient-list/create-edit-list.component.tsx [hunk 1]
```diff
@@ -141,6 +141,7 @@ const CreateEditPatientList: React.FC<CreateEditPatientListProps> = ({
       <div className={styles.input}>
         <Layer level={isTablet ? 1 : 0}>
           <TextArea
+            id="list_description"
```
Comment (anjula-sack) [inline packages/esm-patient-list-app/src/create-edit-patient-list/create-edit-list.component.tsx:144]: Do we need this Id? @Piumal1999

Comment (Piumal1999) [reply packages/esm-patient-list-app/src/create-edit-patient-list/create-edit-list.component.tsx:144]: @anjula-sack For some reason, playwright fails to locate the textarea by label if the id is not defined.

Comment (anjula-sack) [reply packages/esm-patient-list-app/src/create-edit-patient-list/create-edit-list.component.tsx:144]: ah okay

**Review by anjula-sack — APPROVED** [review]
LGTM

**Pull request merged**